src/common/metric.tsx

```diff
@@ -64,9 +65,14 @@ export function getMetricDefinition(metric: Metric) {
   return metricDefinitions[metric];
 }
 
-export function getMetricDisclaimer(metric: Metric, region: Region) {
+export function getMetricDisclaimer(
+  metric: Metric,
+  region: Region,
+  provenanceInfo: MetricToProvenance,
+) {
+  const provenanceForMetric = provenanceInfo[metric];
   const metricDefinition = getMetricDefinition(metric);
-  return metricDefinition.renderDisclaimer(region);
+  return metricDefinition.renderDisclaimer(region, provenanceForMetric);
```

Comment:
`provenanceForMetric` is already metric specific -- `const provenanceForMetric = provenanceInfo[metric]`

but yeah, will pass this down from the top (ChartBlock) instead of passing the whole map from there

Reply:
Oh, haha. My bad.  I'd maybe rename the argument to `provenance` or `source` but feel free to leave as-is if it's clearer to you.